packages/desktop-client/src/components/autocomplete/Autocomplete.tsx

```diff
@@ -559,17 +561,21 @@ function SingleAutocomplete<T extends Item>({
                 )}
               </View>
             ) : (
-              <Tooltip
-                position="bottom-stretch"
+              <Popover
+                triggerRef={triggerRef}
+                placement="bottom start"
                 offset={2}
+                isOpen={isOpen}
+                onOpenChange={close}
+                isNonModal
                 style={{
-                  padding: 0,
+                  ...styles.darkScrollbar,
+                  scrollbarColor: 'rgba(200, 200, 200, .5) rgba(0, 0, 0, 0)',
                   backgroundColor: theme.menuAutoCompleteBackground,
                   color: theme.menuAutoCompleteText,
                   minWidth: 200,
-                  ...tooltipStyle,
+                  width: triggerRef.current?.clientWidth,
                 }}
-                {...tooltipProps}
                 data-testid="autocomplete"
               >
                 {renderItems(
```

Comment:
Maybe we can try wrapping this in a View and setting the scrollbar style there?

Reply:
Would you be open to try this out locally? I'm working blindly here without a way to reproduce.